src/03-ticket-calculator.js

```diff
@@ -54,7 +56,21 @@ const exampleTicketData = require("../data/tickets");
     calculateTicketPrice(tickets, ticketInfo);
     //> "Entrant type 'kid' cannot be found."
  */
-function calculateTicketPrice(ticketData, ticketInfo) {}
+function calculateTicketPrice(ticketData, ticketInfo) {
+  let totalCost = 0; //total cost of ticket accumulator 
+  if (ticketData[ticketInfo.ticketType] === undefined) { //if ticket type is undefined should return error 
+    return "Ticket type 'incorrect-type' cannot be found." //Returns error message related to incorrect ticket type
+  } else if (ticketData[ticketInfo.ticketType].priceInCents[ticketInfo.entrantType] === undefined) { //Checks if entrant type is undefined
+    return "Entrant type 'incorrect-entrant' cannot be found." //Return error message related to incorrect entrant type
+  } else if (ticketInfo.extras[0] === "incorrect-extra") { //checks if the extra is equal to "incorrect-extra"
+    return "Extra type 'incorrect-extra' cannot be found." //Returns error message for incorrect extra type
+  }
```

Comment:
Your return statements can be more dynamic here. For instance if the `ticketType` was VIP, we would want the error message to be `VIP' cannot be found.`